src/components/Dashboard.js

```diff
@@ -4,16 +4,17 @@ import './Dashboard.scss'
 import React, {useEffect, useState} from "react"
 import {getCurrencyValues} from "../services/CurrencyService"
 import {CurrencySelect} from './CurrecySelectComponent/SelectComponent'
+import {inputPlaceholder} from '../constants/Variables'
 
-export const regex = new RegExp("^[0-9]*$")
+export const regex = new RegExp("^[+-]?\\d+(\\.\\d{0,2})?$")
```

Comment:
this const doesn't need to be exported because it's used only inside od this single component. :P

Reply:
You are right. Fixed